src/core/StelProjectorClasses.hpp

```diff
@@ -141,6 +143,8 @@ class StelProjectorCylinder : public StelProjector
 		SphericalCap cap(capN, capD);
 		return cap.intersects(cap1) && cap.intersects(cap2) && cap.intersects(cap3);
 	}
+
+	float objectCenterAzimuth = 0;
```

Comment:
Please add a comment what this is. It is confusing to possible new developers (or for ourselves, after 4 months or so :) that this is always astronomical azimuth (counted from North), regardless of the StelApp.flagAzimuthFromSouth setting, and that this is only used for a particular test (enableOneSideProjection()).